src/CLI/Console.php

```diff
@@ -133,20 +133,24 @@ static public function exit(int $status = 0): void
      */
     static public function execute(string $cmd, string $stdin, string &$stdout, string &$stderr, int $timeout = -1): int
     {
+        $cmd = '( ' . $cmd . ' ) 3>/dev/null ; echo $? >&3';
+
         $pipes = [];
         $process = \proc_open(
             $cmd,
-            [['pipe','r'],['pipe','w'],['pipe','w']],
+            [['pipe','r'],['pipe','w'],['pipe','w'],['pipe', 'w']],
             $pipes
         );
         $start = \time();
         $stdout = '';
         $stderr = '';
+        $statusPipe = '';
```

Comment:
```suggestion
        $status = '';
```

lets just call it status to be consistent with the others....